apps/consent-ui/src/components/layouts/SideImageLayout/SideImageLayout.module.scss

```
@@ -0,0 +1,165 @@
+.container {
+	display: flex;
+	flex-direction: column;
+	width: 100%;
+	@media (min-width: theme('screens.md')) {
+		flex-direction: row;
+	}
+}
+
+.mobileTabletHeader {
+	width: 100%;
+	background-image: linear-gradient(
+		270deg,
+		theme('colors.other.400'),
+		theme('colors.secondary.500')
+	);
+	padding: 1rem;
+	@media (min-width: theme('screens.sm')) {
+		padding: 1.5rem 2.44rem;
+	}
+	@media (min-width: theme('screens.md')) {
+		display: none;
+	}
+}
+
+.title {
+	color: theme('colors.grayscale.100');
+	font-size: 2.25rem;
+	font-weight: 600;
+	line-height: 2.875rem;
```

This is in the config too, should be able to do:
```suggestion
	font-size: theme('fontSize.2xl');
	font-weight: 600;
```